.config/CredScanSuppressions.json

```diff
@@ -4,6 +4,14 @@
         {
             "placeholder": "Aa!1()-xyz",
             "_justification": "This is the secret place holder used by my team"
+        },
```

Comment:
Do we really need to create a couple of different secret?  Can we use one placeholder in all code?

Reply:
Different service has different requirements for secret.